src/Symfony/Component/DependencyInjection/ContainerBuilder.php

```diff
@@ -101,6 +101,11 @@ class ContainerBuilder extends Container implements TaggedContainerInterface
     private $envCounters = array();
 
     /**
+     * @var int A counter to work around buggy internal name generation for anonymous classes.
```

Comment:
See e.g. https://bugs.php.net/73816
See also `zend_generate_anon_class_name` in `Zend/zend_compile.c`.